scipy/optimize/_minimize.py

```diff
@@ -354,9 +349,34 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
     the gradient and the Hessian may be approximated using
     three finite-difference schemes: {'2-point', '3-point', 'cs'}.
     The scheme 'cs' is, potentially, the most accurate but it
-    requires the function to correctly handles complex inputs and to
+    requires the function to correctly handle complex inputs and to
     be differentiable in the complex plane. The scheme '3-point' is more
-    accurate than '2-point' but requires twice as many operations.
+    accurate than '2-point' but requires twice as many operations. If the
+    gradient is estimated via finite-differences the Hessian must be
+    estimated using one of the quasi-Newton strategies.
+
+    **Method specific options for the `hess` keyword**
```

Comment:
This should fix the rendering
```suggestion
    **Method specific options for the** `hess` **keyword**
```

Reply:
The rendering looks ok in the docs though.
https://38591-1460385-gh.circle-artifacts.com/0/html-scipyorg/reference/generated/scipy.optimize.minimize.html?highlight=minimize#scipy.optimize.minimize

Reply:
With the ticks? I thought it should be rendered as a param and not show ticks.

Reply:
RST/sphinx is one of my blindspots. It looks ok to me in the rendered docs, but I don't know any more than that.

Reply:
I will tests quickly locally

Reply:
They're in the notebook that I gave the gist link for. https://gist.github.com/andyfaff/169fe500ccd655d364f623b6de1a824b

Reply:
Thanks, I had a look before. Here I was talking about the doc rendering.

Using single quotes
<img width="382" alt="Screenshot 2021-11-12 at 10 26 00" src="https://user-images.githubusercontent.com/23188539/141444012-bf63e4d1-9875-471a-9f20-5634640e0cf1.png">
double quotes
<img width="384" alt="Screenshot 2021-11-12 at 10 29 24" src="https://user-images.githubusercontent.com/23188539/141444188-5664897e-88b5-40d8-8b36-543a6b42926e.png">

Both is fine for me and better than the tick version.